coding-concepts.js

```diff
@@ -62,7 +62,22 @@ class Learn {
   }
 }
 const learnStudent = new Learn("George")
-// console.log(learnStudent)
-
-// a) Your answer:
-// b) Verify and explain:
+ console.log(learnStudent)
+
+// a) Your answer: I think this will basically replace name in line 59 with George. So the output will be George, Bravo, 2023
```

Comment:
good work on extra content
Since you are attempting to show the output of the method call
`${this.student} is a student in ${this.cohort} in ${this.year}`

George is a student in Bravo in 2023